### PR TITLE
Create model card

### PR DESCRIPTION
**Pull request by mrm8488**

### CODE REVIEW

File: model_cards/mrm8488/t5-base-finetuned-sarcasm-twitter/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+![model image](https://camo.githubusercontent.com/623b4dea0b653f2ad3f36c71ebfe749a677ac0a1/68747470733a2f2f6d69726f2e6d656469756d2e636f6d2f6d61782f343030362f312a44304a31674e51663876727255704b657944387750412e706e67)
```
Comment (julien-c) [inline model_cards/mrm8488/t5-base-finetuned-sarcasm-twitter/README.md:14]: You might want to upload to a more permanent place

**Review by julien-c — APPROVED** [review]

**Comment by julien-c** [comment]
Pretty cool!

File: model_cards/mrm8488/t5-base-finetuned-sarcasm-twitter/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+--
```
Comment (julien-c) [inline model_cards/mrm8488/t5-base-finetuned-sarcasm-twitter/README.md:1]: Oops, there's a `-` missing here (also on https://huggingface.co/mrm8488/t5-base-finetuned-emotion)

Comment (julien-c) [reply model_cards/mrm8488/t5-base-finetuned-sarcasm-twitter/README.md:1]: @mrm8488

**Comment by mrm8488** [comment]
Thank you @julien-c

**Pull request merged**